_posts/randy/2022-07-24-how-video-takes-all-day.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title: "How Video Takes All Day"
+author: randy
+permalink: /@randy/how-video-takes-all-day
+image: "assets/img/how-video-takes-all-day.png"
+description: As video content gets shorter in step with modern attention spans, entertainment value needs to go up.  How do you figure out the creative parts and pieces in short order? Fortunately, thousands of media assets are available online, shaving eons off production time.
+tags: tails randy analog-sampling
```

Comment:
No need for randy tag.  This one gets pulled from the author property

Reply:
old template begone